airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/ConnectorsView.tsx

```diff
@@ -128,13 +129,15 @@ const ConnectorsView: React.FC<ConnectorsViewProps> = ({
       (section === "available" && usedConnectorsDefinitions.length === 0)) && (
       <FlexContainer>
         {allowUploadCustomImage && <CreateConnector type={type} />}
-        <UpgradeAllButton
-          disabled={!((hasNewConnectorVersion || isUpdateSuccess) && allowUpdateConnectors)}
-          isLoading={loading}
-          hasError={!!error && !loading}
-          hasSuccess={isUpdateSuccess}
-          onUpdate={onUpdate}
-        />
+        {!isCloudApp() && (
```

Comment:
```suggestion
        {/* Connectors are automatically kept up-to-date on Cloud */}
        {!isCloudApp() && (
```
Just a small suggestion for a comment explaining why we are hiding this on cloud. I'll leave it up to you to decide if it makes sense to add or if you wanna reword